lms/static/sass/views/_account-settings.scss

```diff
@@ -0,0 +1,107 @@
+.account-settings-container {
+    @extend .container;
+    padding-top: $baseline*2;
```

Comment:
anytime there's a math operation on a css variable, our convention is to wrap these in parentheses. ($baseline*2) I'll note this when I see it on this PR.